### PR TITLE
add luxon by npm install --save luxon

### PR DESCRIPTION
**Pull request by fickryiman**
split classes and function into separates module -> DONE & MERGED
adding luxon date time

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,118 +1,45 @@
...
+const authorInput = document.getElementById('book-author');
 
-function loadBooksInformation() {
-  while (bookContainer.hasChildNodes()) {
-    bookContainer.removeChild(bookContainer.firstChild);
+addForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const title = titleInput.value;
+  const author = authorInput.value;
+  const book = new Book(title, author);
```
Comment (CollinsTatang) [inline index.js]: - [x] Please your app generates an error at line 17 when a user tries to add a new book identifying book as a constructor.
**Check the error message below**
class
Please properly refactor your code.

File: modules/bookClass.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
-export default class Book {
-  constructor(title, author) {
-    this.title = title;
-    this.author = author;
-  }
-}
+/* eslint-disable no-unused-vars */
```
Comment (CollinsTatang) [inline modules/bookClass.js]: Please do not disable your eslint but rather try to fix the errors generated for your linters to pass successfully

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @fickryiman, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (fickryiman) [reply index.js]: has fixes for add book

File: modules/bookClass.js (hunk 2 above)
Comment (fickryiman) [reply modules/bookClass.js]: this function is called in another module

File: modules/bookClass.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
```
Comment (gemmen29) [inline modules/bookClass.js]: - [x] Please remove the `eslint-disable` line as this defies the purpose of using linters in the first place and try to fix the error manually as requested by the previous reviewer. You can fix this error by removing the `let Book;` line as this variable is defined but never used.

File: modules/interfaceClass.js [hunk 4]
```diff
@@ -1,7 +1,10 @@
- import BookCRUD from "./bookCrudClass.js";
- 
- export default class Interface {
-  static renderBooks() {
+import BookCRUD from './bookCrudClass.js';
+
+/* eslint-disable no-unused-vars */
```
Comment (gemmen29) [inline modules/interfaceClass.js]: - [x] Please remove the `eslint-disable` line as this defies the purpose of using linters in the first place and try to fix the error manually as requested by the previous reviewer. You can fix this error by removing the `let Interface;` line as this variable is defined but never used.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @fickryiman, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Highlights💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/bookClass.js (hunk 3 above)
Comment (fickryiman) [reply modules/bookClass.js]: image

File: modules/interfaceClass.js (hunk 4 above)
Comment (fickryiman) [reply modules/interfaceClass.js]: image

File: modules/bookClass.js (hunk 2 above)
Comment (CollinsTatang) [reply modules/bookClass.js]: Yes no problem reusing your function
Remove all unused declare variables

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @fickryiman👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**